python/cuda_parallel/CMakeLists.txt

```
@@ -22,3 +22,22 @@ install(
     TARGETS cccl.c.parallel
     DESTINATION cuda/parallel/experimental/cccl
 )
+
+find_package(Python REQUIRED COMPONENTS Development.Module)
+
+set(CYTHON_FLAGS "-t -w \"${CMAKE_SOURCE_DIR}\"")
+message(STATUS "$ENV{Cython_DIR}")
+message(STATUS "$ENV{Cython_ROOT}")
+
+set(_saved_path ${CMAKE_MODULE_PATH})
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${CMAKE_SOURCE_DIR}/resources/cmake)
```

- Always quote paths, otherwise they'll break if spaces/special chars are used
- Best practice is to use `<project name>_SOURCE_DIR` instead of `CMAKE_SOURCE_DIR`. Otherwise things will break if this project is consumed as part of a larger CMake build.

```suggestion
set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${cuda_parallel_SOURCE_DIR}/resources/cmake")
```